src/main/java/seedu/address/model/student/Student.java

```diff
@@ -82,6 +83,17 @@ public boolean hasTutorial(Tutorial tutorial) {
         return tutorials.contains(tutorial);
     }
 
+    /**
+     * Removes invalid tutorials from the student if it doesn't exist in
+     * {@code validTuts}
+     *
+     * @param validTuts
+     *            Set of valid tutorials
+     */
+    public void removeInvalidTutorials(Set<Tutorial> validTuts) {
+        this.tutorials.removeIf(t -> !validTuts.contains(t));
```

Comment:
```suggestion
        tutorials.removeIf(t -> !validTuts.contains(t));
```
Perhaps avoid unnecessary use of `this` with fields? From coding standard [here](https://se-education.org/guides/conventions/java/index.html#types).